src/index.e2e.test.js

```diff
@@ -46,7 +46,7 @@ describe('index e2e tests', function () {
         }
 
         expectedError.should.exist;
-        expectedError.code.should.equal(2);
+        expectedError.code.should.equal(1);
```

Comment:
I think this should stay as exit code 2 seen in the read me:

Exit Code 1: Not all arguments have been provided

Exit Code 2: No availability in the range provided